File: packages/cli/test/dev/fixtures/services-cron-js/cron/task.js
const fs = require('node:fs');
const path = require('node:path');

const RESULT_DIR = path.join(__dirname, '..', '.results');

module.exports = async function runCronTask() {
  fs.mkdirSync(RESULT_DIR, { recursive: true });
  fs.writeFileSync(
    path.join(RESULT_DIR, 'cron_result.json'),
    JSON.stringify({ executed: true })
  );
};

File: packages/cli/test/dev/fixtures/services-cron/cron/task.py
import json
import os

RESULT_DIR = os.path.join(os.path.dirname(os.path.dirname(__file__)), ".results")


def run_cron_task():
    os.makedirs(RESULT_DIR, exist_ok=True)
    with open(os.path.join(RESULT_DIR, "cron_result.json"), "w") as f:
        json.dump({"executed": True}, f)
